ios/Exponent/Versioned/Modules/Api/EXNotifications.m

```diff
@@ -40,6 +41,18 @@ - (void)setBridge:(RCTBridge *)bridge
   _experienceId = _bridge.experienceScope.experienceId;
 }
 
+
+RCT_REMAP_METHOD(getDevicePushTokenAsync,
+                 getDevicePushTokenAsyncWithResolver:(RCTPromiseResolveBlock)resolve
+                 rejecter:(RCTPromiseRejectBlock)reject)
+{
+  NSData *token = [[NSUserDefaults standardUserDefaults] apnsToken];
+  if (!token) {
+    return reject(0, @"APNS token has not been set", nil);
+  }
+  return resolve([token apnsTokenString]);
```

Comment:
I think we should resolve in a format more like `{ type: "apns", data: "someToken" }` since the token format will differ depending on platform and underlying API.